sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_recognize_receipts_async.py

```diff
@@ -46,18 +46,38 @@ async def recognize_receipts(self):
+                transaction_date = receipt.fields.get("TransactionDate")
+                if transaction_date:
+                    print("Transaction Date: {} has confidence: {}".format(transaction_date.value, transaction_date.confidence))
                 print("Receipt items:")
-                for item in receipt.receipt_items:
-                    print("...Item Name: {} has confidence: {}".format(item.name.value, item.name.confidence))
-                    print("...Item Quantity: {} has confidence: {}".format(item.quantity.value, item.quantity.confidence))
-                    print("...Individual Item Price: {} has confidence: {}".format(item.price.value, item.price.confidence))
-                    print("...Total Item Price: {} has confidence: {}".format(item.total_price.value, item.total_price.confidence))
-                print("Subtotal: {} has confidence: {}".format(receipt.subtotal.value, receipt.subtotal.confidence))
-                print("Tax: {} has confidence: {}".format(receipt.tax.value, receipt.tax.confidence))
-                print("Tip: {} has confidence: {}".format(receipt.tip.value, receipt.tip.confidence))
-                print("Total: {} has confidence: {}".format(receipt.total.value, receipt.total.confidence))
+                for item in receipt.fields.get("Items").value:
+                    item_name = item.value.get("Name")
+                    if item_name:
+                        print("...Item Name: {} has confidence: {}".format(item_name.value, item_name.confidence))
+                    item_quantity = item.value.get("Quantity")
+                    if item_quantity:
+                        print("...Item Quantity: {} has confidence: {}".format(item_quantity.value, item_quantity.confidence))
+                    item_price = item.value.get("Price")
+                    if item_price:
+                        print("...Individual Item Price: {} has confidence: {}".format(item_price.value, item_price.confidence))
+                    item_total_price = item.value.get("TotalPrice")
+                    if item_total_price:
+                        print("...Total Item Price: {} has confidence: {}".format(item_total_price.value, item_total_price.confidence))
+                subtotal = receipt.fields.get("Subtotal")
+                if subtotal:
+                    print("Subtotal: {} has confidence: {}".format(subtotal.value, subtotal.confidence))
+                tax = receipt.fields.get("Tax")
+                if tax:
+                    print("Tax: {} has confidence: {}".format(tax.value, tax.confidence))
+                tip = receipt.fields.get("Tip")
+                if tip:
+                    print("Tip: {} has confidence: {}".format(tip.value, tip.confidence))
+                total = receipt.fields.get("Total")
+                if total:
+                    print("Total: {} has confidence: {}".format(total.value, total.confidence))
```

Comment:
I understand wanting to show the keys available, but wondering if we should do at least one sample or readme like this:

```
for receipt in result:
    for key, val in receipt.fields.items():
        print("{}: {} has confidence: {}".format(key, val.value, val.confidence))
```

Reply:
I see what you mean. I think I will leave the samples like this, but update the readme sample to traverse the fields like this

Reply:
I really like how the readme looks. 
For this sample, I will suggest adding a comment on top a [link ](https://westus2.dev.cognitive.microsoft.com/docs/services/form-recognizer-api-v2-preview/operations/GetAnalyzeReceiptResult) to the types the service is currently supporting, or guiding the user into finding this information

Reply:
For the .NET sample, we're only showing a subset of illustrative values from the receipt.

Also, I'm not sure how it works in Python, but do you need to check the FieldValueType  to cast these to the right time in order for a customer to use them in a scenario?